echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/BearychatNotificationAgent.groovy

```diff
@@ -49,52 +49,47 @@ class BearychatNotificationAgent extends AbstractEventNotificationAgent {
 
   @Override
   void sendNotifications(Map preference, String application, Event event, Map config, String status) {
-    try {
-      String buildInfo = ''
-
-      if (config.type == 'pipeline' || config.type == 'stage') {
-        if (event.content?.execution?.trigger?.buildInfo?.url) {
-          buildInfo = """build #<a href="${event.content.execution.trigger.buildInfo.url}">${
-            event.content.execution.trigger.buildInfo.number as Integer
-          }</a> """
-        }
-      }
-      log.info('Sending bearychat message {} for {} {} {} {}', kv('address', preference.address), kv('application', application), kv('type', config.type), kv('status', status), kv('executionId', event.content?.execution?.id))
-
+    String buildInfo = ''
 
-      String message = ''
-
-      if (config.type == 'stage') {
-        String stageName = event.content.name ?: event.content?.context?.stageDetails?.name
-        message = """Stage $stageName for """
+    if (config.type == 'pipeline' || config.type == 'stage') {
```

Comment:
Pulling on a thread here... I think we may want to `isExecution(config.type)` here as well. I bet the other notification agents also have similar conditions.

Reply:
how about I save those improvements for going through and cleaning up all the notification logic?